boards/arm/nucleo_wl55jc/arduino_r3_connector.dtsi

```diff
@@ -36,3 +36,4 @@
 };
 
 arduino_serial: &lpuart1 {};
+arduino_i2c: &i2c2 {};
```

Comment:
Matching i2c2 node not supported in dtsi.

Reply:
Oops, I accidently remove it while fixing buidkite issue.
I2C2, I2C3 and SPI2 restored.